src/port/ui/PortMenu.cpp

```diff
@@ -410,6 +410,12 @@ void PortMenu::AddEnhancements() {
         })
         .Options(UIWidgets::CheckboxOptions({ { .tooltip = "Edit the universe!" } }));
 #endif
+
+    path = { "Enhancements", "Bugfixes", SECTION_COLUMN_1 };
+    AddSidebarEntry("Enhancements", "Bugfixes", 3);
+    AddWidget(path, "Don't reset random seed", WIDGET_CVAR_CHECKBOX)
```

Comment:
`Fix random seeding` ? Maybe `Prevent random seeding from being reset`
Or perhaps
`Randomize racer spawns` and then have the explanation in the tooltip?